avocado/core/__init__.py

```diff
@@ -178,6 +178,18 @@ def register_core_options():
         help_msg=help_msg,
```

Comment:
The commit message looks like it needs updating.  AFAICT from the patch itself, it's a different config option rather than a special kwarg to introduce the environment clearing.

Reply:
thank you for catching it. I need to update the commit message to correspond the latest changes.